blog/2022-06-30-paralus-quickstart-aks/index.md

```diff
@@ -0,0 +1,198 @@
+
+   NAME: myrelease
+   LAST DEPLOYED: Wed Jun 29 10:13:48 2022
+   NAMESPACE: paralus
+   STATUS: deployed
+   REVISION: 1
+   NOTES:
+   1. Access the application URL by running these commands:
+     Get load balancer address via:
+     kubectl get service envoy --namespace paralus -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'
+
+     Add DNS records of following domains such that it resolves to above address:
+     - console-aks-oss.chartexample.com
+     - *.core-connector.aks-oss.chartexample.com
+     - *.user.aks-oss.chartexample.com
+     Open http://console-aks-oss.chartexample.com in browser.
+
+   You can view the recovery link for admin user by running the following command once all the pods are running:
+
+   kubectl logs -f --namespace paralus $(kubectl get pods --namespace paralus -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize-paralus | grep 'Org Admin signup URL:'
+   ```
+
+   > Note: It can take upto a few minutes before all the pods are running and you can access the dashboard. You can check the status using `watch kubectl get pods`
+
+## Configuring DNS Settings
+
+Once the installation is complete, you need to first get the external IP address provided by Azure loadbalancer. You can do so by executing the following command:
+
+```bash
+kubectl get svc -n paralus
+
+NAME                            TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                         AGE
+dashboard                       ClusterIP      10.0.134.169   <none>         80/TCP                          3m13s
+elasticsearch-master            ClusterIP      10.0.125.222   <none>         9200/TCP,9300/TCP               3m13s
+elasticsearch-master-headless   ClusterIP      None           <none>         9200/TCP,9300/TCP               3m13s
+myrelease-contour               ClusterIP      10.0.151.61    <none>         8001/TCP                        3m13s
+myrelease-contour-envoy         LoadBalancer   10.0.33.6      13.71.51.105   80:30193/TCP,443:30873/TCP      3m13s
+myrelease-kratos-admin          ClusterIP      10.0.16.90     <none>         80/TCP                          3m13s
+myrelease-kratos-courier        ClusterIP      None           <none>         80/TCP                          3m13s
+myrelease-kratos-public         ClusterIP      10.0.234.235   <none>         80/TCP                          3m13s
+myrelease-postgresql            ClusterIP      10.0.114.76    <none>         5432/TCP                        3m13s
+myrelease-postgresql-hl         ClusterIP      None           <none>         5432/TCP                        3m13s
+paralus                         ClusterIP      10.0.12.103    <none>         11000/TCP,10000/TCP,10001/TCP   3m13s
+prompt                          ClusterIP      10.0.50.46     <none>         7009/TCP                        3m13s
+relay-server      
+```
+
+Note down the IP address for the service type `LoadBalancer`
```

Comment:
Should we be explicit about it being the `externall-ip` of `<releasename>-contour-envoy`?

Reply:
In case of AWS, we did get a [fqdn instead of external IP](https://www.paralus.io/blog/eks-quickstart#configuring-dns-settings), however in case of Azure got the IP address which should be configured in DNS settings.

Reply:
What I meant was we could change it  to be explicit instead of just saying get the ip of the LB.
```suggestion
Note down the `EXTERNAL-IP` address for the `<releasename>-contour-envoy` service.
```